frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -1,39 +1,133 @@
+      </Flex>
+      <Text lh={"1.25rem"} mb={"lg"}>
+        {description}
+      </Text>
+      <Flex gap="lg" direction="column" align="flex-start">
+        {children}
+      </Flex>
+    </StyledCard>
   );
 }
+
+export const StaticEmbeddingOptionCard = () => {
+  const enabled = useSelector(state => getSetting(state, "enable-embedding"));
+  const upgradeUrl = useSelector(state =>
+    getUpgradeUrl(state, { utm_media: "embed_standalone" }),
+  );
+  const shouldPromptToUpgrade = !PLUGIN_EMBEDDING.isEnabled();
+
+  const upgradeText = jt`A "powered by Metabase" banner appears on static embeds. You can ${(
+    <ExternalLink key="upgrade-link" href={upgradeUrl}>
+      {t`upgrade to a paid plan`}
+    </ExternalLink>
+  )} to remove it.`;
+
+  return (
+    <EmbeddingOption
+      icon={enabled ? <StaticEmbeddingOn /> : <StaticEmbeddingOff />}
+      title={t`Static embedding`}
+      description={jt`Use static embedding when you don’t want to give people ad hoc query access to their data for whatever reason, or you want to present data that applies to all of your tenants at once.${
+        shouldPromptToUpgrade && (
+          <Text size="sm" mt="xs" key="upgrade-text">
+            {upgradeText}
+          </Text>
+        )
+      }`}
+    >
+      <Button
+        variant="default"
+        disabled={!enabled}
+        component={Link}
+        to={"/admin/settings/embedding-in-other-applications/standalone"}
+      >
+        {t`Manage`}
+      </Button>
+    </EmbeddingOption>
+  );
+};
+
+export const InteractiveEmbeddingOptionCard = () => {
+  const isEE = PLUGIN_EMBEDDING.isEnabled();
+  const enabled = useSelector(state => getSetting(state, "enable-embedding"));
+
+  return (
+    <EmbeddingOption
+      icon={enabled ? <InteractiveEmbeddingOn /> : <InteractiveEmbeddingOff />}
+      title={t`Interactive embedding`}
+      label={t`PRO & ENTERPRISE`}
+      description={jt`Use interactive embedding when you want to ${(
+        <ExternalLink
+          href="https://www.metabase.com/blog/why-full-app-embedding"
+          key="why-full-app-embedding"
+        >
+          {t`offer multi-tenant, self-service analytics`}
+        </ExternalLink>
+      )} and people want to create their own questions, dashboards, models, and more, all in their own data sandbox.`}
+    >
+      <BoldExternalLink
+        href={
+          isEE
+            ? interactiveEmbedQuickStartEELink
+            : interactiveEmbedQuickStartOSSLink
+        }
+      >
+        {t`Check out our Quick Start`}
+      </BoldExternalLink>
+      {isEE ? (
+        <Button
+          component={Link}
+          to={"/admin/settings/embedding-in-other-applications/full-app"}
+          disabled={!enabled}
+        >
+          {t`Configure`}
+        </Button>
+      ) : (
+        <Button
+          component={ExternalLink}
+          href="https://www.metabase.com/product/embedded-analytics?utm_source=product&utm_medium=CTA&utm_campaign=embed-settings-oss-cta"
```

Comment:
My rationale was that I extracted those two because they were long and only differed in 3 letters and I didn't like a ternary like this (without line wrap they look the same).
```
      <BoldExternalLink
        href={
          isEE
            ? "https://www.metabase.com/learn/customer-facing-analytics/interactive-embedding-quick-start?utm_source=product&utm_medium=CTA&utm_campaign=embed-settings-pro-cta"
            : "https://www.metabase.com/learn/customer-facing-analytics/interactive-embedding-quick-start?utm_source=product&utm_medium=CTA&utm_campaign=embed-settings-oss-cta"
        }
      >
```
I usually inline everything that's used only once though, so that creates the inconsistency.
If you don't find it that bad I could inline those two too, what do you think?

Reply:
Inlining isn't that bad here.

Reply:
🙏 
 [6c0c430047e51dc15028437dcfdf71fb282709a5](https://github.com/metabase/metabase/commit/6c0c430047e51dc15028437dcfdf71fb282709a5)